camunda-formio-tasklist-vue/src/components/TaskList.vue

```diff
@@ -25,29 +25,33 @@
         />
       </b-col>
       <b-col v-if="getFormsFlowTaskId && task" :lg="maximize ? 9 : 12" md="12">
-        <ExpandContract />
         <div class="cft-service-task-details">
-          <b-row
-            class="ml-0 cft-task-header task-header-title"
-            v-b-tooltip.hover
-            title="Task Name"
-          >
-            {{ task.name }}
+          <b-row>
```

Comment:
The text for TaskName, ProcessName and applicationId doesn't look alligned. Would it be an issue to the client?

![image](https://user-images.githubusercontent.com/70306694/125152434-1e88d900-e16a-11eb-96c1-63eff6caeb2e.png)